pyproject.toml

```diff
@@ -121,12 +121,8 @@ lint.ignore = [
     "ANN002", # TODO: missing-type-args
     "ANN003", # TODO: missing-type-kwargs
     "ANN201", # TODO: missing-return-type-undocumented-public-function
-    "COM812", # missing-trailing-comma (ruff-format recommended)
     "D203", # one-blank-line-before-class
     "D212", # blank-line-before-class
-    "ERA001", # TODO: commented-out-code
-    "ISC001", # single-line-implicit-string-concatenation (ruff-format recommended)
```

Comment:
You have to leave this one in, that is why I didn't write TODO... In fact, there are more than I thought - just found the docs https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules

Reply:
Could you make an issue to look into these conflicting rules?